projects/m2k/system_bd.tcl

```diff
@@ -1,17 +1,41 @@
 ###############################################################################
-## Copyright (C) 2017-2024 Analog Devices, Inc. All rights reserved.
+## Copyright (C) 2014-2025 Analog Devices, Inc. All rights reserved.
 ### SPDX short identifier: ADIBSD
 ###############################################################################
 
+set_msg_config -id {PSU-1} -new_severity {WARNING}
+
 add_files -norecurse  $ad_hdl_dir/library/util_cdc/sync_bits.v
 add_files -norecurse  $ad_hdl_dir/library/util_cdc/sync_event.v
 
+set DEBUG_BUILD 0
 if {[info exists DEBUG_BUILD] == 0} {
```

Comment:
Remove this lines. It does not make sense. Defineing DEBUG_BULD and afterwards check that it exists. Leave only the set to 0.